_projects/editor_support.md

```diff
@@ -1,11 +1,9 @@
 ---
 collaborating_projects:
   - coala
-status: in_progress
 desc: "Develop an efficient strategy that allows coala to maintain a number of plugins for different editors."
 difficulty: medium
 initiatives:
-  - GSoC
```

Comment:
please re-add GSoC here.

What I meant by 'non-GSoC' was that it had been marked as 'in progress', but no gsoc project had ever commenced, and I wasnt aware of who might have been doing it.

I've double checked now, and this project is certainly available to be worked on.